src/docs/security-and-privacy-compliance/devops-security-considerations.md

```diff
@@ -1,300 +1,276 @@
 **OpenShift Secrets:**
 
 This 'secrets' store should actually only be used for configurations.  Values are encoded as base64 and NOT encrypted.  However, these 'secrets' can only be accessed with a role to a given namespace with permission to access them.  Additionally, the physical etcd volume, where OpenShift secrets are stored, is encrypted.
 
 **Vault:**
+Vault is the preferred secrets management tool to use on OpenShift.
 
-The preferred secrets management tool for team use on OpenShift.
-- https://cloud.gov.bc.ca/private-cloud/our-products-in-the-private-cloud-paas/vault-secrets-management/
-- [Vault secrets management service](/vault-secrets-management-service/)
-- [Vault getting started guide](/vault-getting-started-guide/)
+* Find out more about the [benefits of using Vault](https://digital.gov.bc.ca/cloud/services/private/products-tools/vault/)
+* [Vault secrets management service](/vault-secrets-management-service/)
+* [Vault getting started guide](/vault-getting-started-guide/)
 
-------
-### <a name="gitops-cluster-configuration-management"></a>GitOps/Cluster Configuration Management
+## GitOps/Cluster configuration management
 
-Argo CD provides a GitOps capability for sync'ing a Git repository with an OpenShift configuration (platform or application).
-- [Argo CD usage](/argo-cd-usage/)
+Argo CD (integrated into OpenShift as the GitOps Operator) provides a GitOps capability for sync'ing a Git repository with an OpenShift configuration (platform or application)
 
-We are in the process of testing out the GitOps Operator, based on ArgoCD, that is integrated into OpenShift.  This may replace (partially or completely) our custom ArgoCD implementation.
+## Application Programmable Interface (API) management
 
-------
-### <a name="api-management"></a>Application Programmable Interface (API) Management
+The Data BC team hosts an API Gateway for use by other government clients. 
 
-The Data BC team hosts an API Gateway for use by other government clients.  Details can be found here:
-- https://developer.gov.bc.ca/API-Gateway-(powered-by-Kong-CE)
-- https://developer.gov.bc.ca/BC-Government-API-Guidelines
-- https://developer.gov.bc.ca/BC-Government-OpenAPI-Specifications
+Details can be found here:
 
-------
-### <a name="application-resource-tuning"></a>Application Resource Tuning Advisor and App Assessment Tool
-**Resource Tuning Advisor**
-- https://github.com/BCDevOps/resource-tuning-advisor-app 
+* [B.C. government API Program Services](https://www2.gov.bc.ca/gov/content/data/about-data-management/databc/databc-products-services/api-management)
+* [B.C. government API guidelines](https://developer.gov.bc.ca/BC-Government-API-Guidelines)
 
-**App Assessment Tool**
-- https://github.com/bcgov/AppAssessment
-
-------
-### <a name="logging-monitoring"></a>Logging/Monitoring (EKS, Kibana, Graphana, Sysdig Monitor, SIEM, Uptime, Status)
+## Logging and monitoring (EKS, Kibana, Graphana, Sysdig Monitor, SIEM, Uptime, Status)
```

Comment:
Unsure what EKS is? Is that supposed to be EFK for ElasticSearch/FluentD/Kibana ?

Reply:
Updated.  Thanks.